dtests/src/test/java/io/snappydata/hydra/misc/SNAP-2269-2762.conf

```diff
@@ -0,0 +1,22 @@
+INCLUDE $JTESTS/io/snappydata/hydra/northwind/startEmbeddedModeCluster.conf;
+
+INITTASK    taskClass   = io.snappydata.hydra.misc.MiscTest taskMethod  = HydraTask_verify_snap2269_snap2762
+    threadGroups = leadThreads
+    ;
+
+INCLUDE $JTESTS/io/snappydata/hydra/northwind/stopEmbeddedModeCluster.conf;
+
+/* end task must stop snappy members because they are not stopped by Hydra */
+hydra.Prms-alwaysDoEndTasks = true;
+hydra.Prms-totalTaskTimeSec = 120;
+hydra.Prms-maxResultWaitSec = 300;
+
+hydra.VmPrms-extraVMArgs   += fcn "hydra.TestConfigFcns.duplicate
```

Comment:
Verify and check the configuration for hydra client VMs. Reduce the VM size in case 1 GB not needed.